sdk/communication/communication-sms/CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 # Release History
 
+## 1.0.1 (Unreleased)
+
+
 ## 1.0.0 (2021-03-29)
 
 - Stable release of `@azure/communication-sms`.
 
+## 1.0.0-beta.3 (2020-11-16)
```

Comment:
@beltr0n The automation moved these entries up because the date on 1.0.0-beta.4 is pointing to 2020 instead of 2021. Can you push a commit to fix this please?